src/types/packager.go

```diff
@@ -6,6 +6,9 @@ package types
 
 // PackagerConfig is the main struct that the packager uses to hold high-level options.
 type PackagerConfig struct {
+	//ZarfCommonOptions inline common options
```

Comment:
Is there a reason this needs to be included here? We kept it in `config.CommonOptions` intentionally as it was shared amongst all instance of an execution (in case we ever have more than one package operation per run).

Reply:
Agree. Changes are in https://github.com/defenseunicorns/zarf/pull/1191/commits/6b1930dc8ef0850dffb6a635844e49b7e6bd44dd